linux/xkb/symbols/3l

```diff
@@ -78,3 +78,16 @@ xkb_symbols "cros" {
         symbols[Group1] = [ Tab ]
     };
 };
+
+// A variant for emacs users, which maps control to caps lock and (re)maps 
+// tab back to tab.
+partial modifier_keys
+xkb_symbols "3l-emacs" {
+    include "us(3l)"
```

Comment:
`3l(basic)`

This is correct for upstream xkeyboard-config version though, the file name just differs for this copy.

Reply:
Oops! Sorry about the copy/paste